accumulo_plugin/plugin-accumulo-2.0/PelzCryptoService.java

```diff
@@ -52,30 +52,48 @@
 import org.apache.accumulo.core.crypto.streams.BlockedOutputStream;
 import org.apache.accumulo.core.crypto.streams.DiscardCloseOutputStream;
 import org.apache.accumulo.core.crypto.streams.RFileCipherOutputStream;
-import org.apache.accumulo.core.cryptoImpl.NoFileDecrypter;
 import org.apache.accumulo.core.spi.crypto.CryptoEnvironment;
 import org.apache.accumulo.core.spi.crypto.CryptoService;
 import org.apache.accumulo.core.spi.crypto.FileDecrypter;
 import org.apache.accumulo.core.spi.crypto.FileEncrypter;
+import org.apache.accumulo.core.spi.crypto.NoFileDecrypter;
+import org.apache.accumulo.core.spi.crypto.NoFileEncrypter;
 import org.apache.commons.io.IOUtils;
 
 /*
  * Example implementation of AES encryption for Accumulo
  */
 public class PelzCryptoService implements CryptoService {
-
+  // properties required for using this service
+  private static final String CRYPTO_PREFIX = "instance.crypto.opts.";
+  private static final String KEY_URI = CRYPTO_PREFIX + "key.uri";
+  // optional properties
+  // defaults to true
+  private static final String ENCRYPT_ENABLED = CRYPTO_PREFIX + "enabled";
+	
   // Hard coded NoCryptoService.VERSION - this permits the removal of NoCryptoService from the
   // core jar, allowing use of only one crypto service
   private static final String NO_CRYPTO_VERSION = "U+1F47B";
 
+  //Version Strings for AESCryptoService
+  private static final String AES_GCM_CRYPTO_VERSION = "U+1F43B";
```

Comment:
The PelzCryptoSerivce also uses AES-GCM and AES-CBC. You might consider renaming these something like:  

    AES_CRYPTO_SERVICE_GCM_VERSION = "U+14F3B";